lib/parse.js

```diff
@@ -84,6 +135,35 @@ var INLINE_CODE = 'inlineCode';
 var BREAK = 'break';
 var ROOT = 'root';
 
+/** TODO */
+function isWhiteSpace(character) {
```

Comment:
:+1: That's really not a big deal. I just wanted to share some thoughts. This effort on rewriting the parser is way more important for the project!